packages/sdk/src/middleware/middleware.ts

```diff
@@ -37,7 +37,17 @@ const actionToUri = (action: TSdkAction, projectKey?: string): string => {
   const service = requestBuilder[action.payload.service];
   if (action.payload.options) service.parse(action.payload.options);
 
-  return service.build();
+  return service.build({
+    // given `service=orderEdits` and given `applyOrderEditTo`, we build an apply endpoint
+    // given `service=orderEdits` and no `applyOrderEditTo`, we build an update endpoint
+    // https://docs.commercetools.com/api/projects/order-edits
+    applyOrderEdit:
+      action.payload.service === 'orderEdits' &&
```

Comment:
This is a bit hacky, but the request-builder doesn't expose any type that I can make use of.